app/src/androidTest/java/org/mozilla/focus/activity/BrowsingHistoryTest.java

```diff
@@ -154,31 +154,17 @@ public void browsingTwoWebSites_sitesAreDisplayedInOrderInHistoryPanel() {
         onView(withId(R.id.browsing_history_recycler_view))
                 .check(matches(atPosition(2, hasDescendant(withText(targetUrlSite1)))));
 
-    }
-
-    /**
-     * Test case no: his_002
-     * Test case name: Visit a page via history record
-     * Steps:
-     * 1. Follow steps in his_001
-     * 2. Tap on a history
-     * 3. Navigate to the selected history site */
-    @Test
-    public void browsingWebsiteThenTapTheFirstHistoryItem_browsingHistoryRecordCorrectly() {
-
-        final String targetUrl = browsingWebsiteAndOpenHistoryPanel();
-
         // Click the first item in history panel, index 0 is date label so index 1 is the latest one of history item
         onView(ViewMatchers.withId(R.id.browsing_history_recycler_view))
-                .perform(RecyclerViewActions.actionOnItemAtPosition(1, click()));
+                .perform(RecyclerViewActions.actionOnItemAtPosition(2, click()));
```

Comment:
not sure why it's 2 here. If that's really what you want,  change the comment above as well.

Reply:
yes it is what i need because I added two history item in this testing, so the last history index is 2. I will change comment accordingly, thanks